$OpenBSD: patch-ghc_mk,v 1.10 2015/01/01 19:25:36 kili Exp $

Fix the bindist-list (for building the bootstrapper); without this,
gtar creates an archive which our tar can't extract.

--- ghc.mk.orig	Thu Jul 10 06:27:16 2014
+++ ghc.mk	Sun Nov  2 21:50:31 2014
@@ -1007,6 +1007,8 @@ unix-binary-dist-prep:
 	echo "UseSystemLibFFI    = $(UseSystemLibFFI)"    >> $(BIN_DIST_MK)
 	cd $(BIN_DIST_PREP_DIR) && autoreconf
 	$(call removeFiles,$(BIN_DIST_PREP_TAR))
+	sed 's@/\./@/@g' bindist-list | sort -u > bindist-list.new
+	mv bindist-list.new bindist-list
 # h means "follow symlinks", e.g. if aclocal.m4 is a symlink to a source
 # tree then we want to include the real file, not a symlink to it
 	cd bindistprep && "$(TAR_CMD)" hcf - -T ../bindist-list | $(TAR_COMP_CMD) -c > ../$(BIN_DIST_PREP_TAR_COMP)
